app/views/layouts/_header.html.slim

```diff
@@ -12,7 +12,7 @@
     = header.with_navigation_item text: t("nav.create_a_job_alert"), href: new_subscription_path
     = header.with_navigation_item text: t("nav.your_profile"), href: jobseekers_profile_path
     = header.with_navigation_item text: t("nav.your_account"), href: jobseekers_account_path, active: your_account_active?
-    = header.with_navigation_item text: t("nav.sign_out"), href: destroy_jobseeker_session_path, options: { method: :delete }
+    = header.with_navigation_item text: t("nav.sign_out"), href: jobseeker_logout_uri.to_s
```

Comment:
This is an existing helper previously built for OneLogin. It builds the URL for onelogin `/logout` endpoint with the callback `/sign_out` endpoint in our side to be redirected back to.